src/fides/api/models/privacy_notice.py

```diff
@@ -178,15 +179,6 @@ class PrivacyNoticeBase:
     displayed_in_api = Column(Boolean, nullable=False, default=False)
     notice_key = Column(String, nullable=False)
 
-    # Attribute that can be temporarily cached as the result of "get_related_privacy_notices"
-    # for a given user, for surfacing CurrentPrivacyPreferences for the user.
-    current_preference: Optional[str] = None
-    outdated_preference: Optional[str] = None
-    # Attributes that can be temporarily cached on the notice to see if the most
-    # recent version or a previous version of a notice have ever been served to the user
-    current_served: Optional[bool] = None
-    outdated_served: Optional[bool] = None
-
```

Comment:
Discussed elsewhere, removed, related to how we used to supplement the Experience with previously-saved preferences.